server/src/main/scala/org/labrad/browser/NodeApi.scala

```diff
@@ -50,6 +50,27 @@ class NodeApi(cxn: LabradConnection)(implicit ec: ExecutionContext) extends Logg
     }
   }
 
+  def autostartNode(node: String): Future[String] = {
+    cxn.to(node).call("autostart").map { _ =>
+      "autostarted"
+    }
+  }
+
+  def autostartList(node: String): Future[Seq[String]] = {
+    async {
+      val list = await { cxn.to(node).call("autostart_list") }
+      list.get[Seq[String]]
+    }
+  }
+
+  def autostartAdd(node: String, server: String): Unit = {
```

Comment:
It would be better to make the return type `Future[Unit]` so that we don't reply to the client until the call to the node is complete, and we send an error back to the client if the call to the node fails.